src/collections/_documentation/development/server/environment.md

```diff
@@ -1,57 +1,38 @@
 ---
 title: 'Development Environment'
-sidebar_order: 1
+sidebar_order: 2
 ---
 
-## Setting up Your Development Environment
+This guide will step you through configuring a local development environment for the Sentry server on macOS. If you're using another operating system the instructions are still roughly the same, but we don't maintain any official documention.
 
-### Linux
+#### A quick note on Bash vs Zsh...
 
-Follow [the Docker file in the repository](https://github.com/getsentry/sentry/blob/master/docker/Dockerfile) but instead of creating and installing a wheel, install the package in editable mode (`pip install . -e`).
+macOS Catalina will recommend that you use `zsh` instead of `bash`. While the differences are minimal, this guide primarily covers bash. You can verify your shell by typing `echo $SHELL`.
```

Comment:
backtick `bash` at end of sentence for consistency?

"Verify your shell ..."

Reply:
bash is both a command (thus `bash`) as well as program. doesn't need to be consistently in backticks (though probably should be Bash vs bash)